docs/concepts-advanced/environment-variables.md

```diff
@@ -1,170 +1,352 @@
+#### Organization
 
-### Runtime Environment Variables (Lagoon API)
+Organization env vars are set for all environments, in all projects, in an organization. They can be overridden by project and environment env vars.
 
-Runtime environment variables are automatically made available in all containers, but they are only added or updated after an environment has been re-deployed.
+An example of setting an `organization` env var:
 
-This defines a project wide runtime variable (available in all environments) for the project with ID `463`:
+=== "Lagoon CLI"
 
-```graphql title="Add runtime variable"
-mutation addRuntimeEnv {
-  addEnvVariable(
-    input:{
-      type:PROJECT,
-      typeId:463,
-      scope:RUNTIME,
-      name:"MYVARIABLENAME",
-      value:"MyVariableValue"
+    ```bash title=""
+    lagoon add variable --organization-name lagoon-demo-org --scope runtime \
+    --name SSMTP_MAILHUB --value mailhub.example.com
+    ```
+
+=== "Lagoon API"
+
+    ```graphql title=""
+    mutation {
+      addOrUpdateEnvVariableByName(
+        input: {
+          organization: "lagoon-demo-org"
+          scope: RUNTIME
+          name: "SSMTP_MAILHUB"
+          value: "mailhub.example.com"
+        }
+      ) {
+        id
+        name
+        scope
+        value
+      }
     }
-  ) {
-    id
-  }
-}
-```
+    ```
+
+#### Project
+
+Project env vars are set for all environments in a project. They can be overridden by environment env vars.
+
+An example of setting a `project` env var:
+
+=== "Lagoon CLI"
+
+    ```bash title=""
+    lagoon add variable --project lagoon-demo --scope runtime \
+    --name PHP_MEMORY_LIMIT --value 400M
+    ```
+
+=== "Lagoon API"
+
+    ```graphql title=""
+    mutation {
+      addOrUpdateEnvVariableByName(
+        input: {
+          project: "lagoon-demo"
+          scope: RUNTIME
+          name: "PHP_MEMORY_LIMIT"
+          value: "400M"
+        }
+      ) {
+        id
+        name
+        scope
+        value
+      }
+    }
+    ```
+
+#### Environment
+
+Environment env vars are set for a single Lagoon environment.
+
+An example of setting an `environment` env var:
+
+=== "Lagoon CLI"
+
+    ```bash title=""
+    lagoon add variable --project lagoon-demo --environment dev \
+    --scope runtime --name LOG_LEVEL --value DEBUG
+    ```
+
+=== "Lagoon API"
+
+    ```graphql title=""
+    mutation {
+      addOrUpdateEnvVariableByName(
+        input: {
+          project: "lagoon-demo"
+          environment: "dev
+          scope: RUNTIME
+          name: "LOG_LEVEL"
+          value: "DEBUG"
+        }
+      ) {
+        id
+        name
+        scope
+        value
+      }
+    }
+    ```
+
+### Scopes
+
+Scopes are used to set an env var for a specific context. An env var can have a scope of `buildtime`, `runtime`, `global`, `container_registry` or `internal_container_registry`.
+
+#### Buildtime
```

Comment:
Shouldn't this be `build` since that is what the scope is called?

Reply:
I wrote it that way originally, but found it cumbersome to read/write. I much prefer "buildtime" for just normal documentation language and it matches "runtime." I got a little carried away with the find/replace, will fix up the places where the api scope name is used.